[2024-01-30 11:29:19 +0800] [4981] [INFO] Starting gunicorn 21.2.0
[2024-01-30 11:29:19 +0800] [4981] [INFO] Listening at: http://0.0.0.0:9394 (4981)
[2024-01-30 11:29:19 +0800] [4981] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 11:29:19 +0800] [4982] [INFO] Booting worker with pid: 4982
[2024-01-30 11:29:19 +0800] [4983] [INFO] Booting worker with pid: 4983
[2024-01-30 11:29:19 +0800] [4982] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 5, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from gvm_server.client import get_gvm_conn
ModuleNotFoundError: No module named 'gvm_server'
[2024-01-30 11:29:19 +0800] [4982] [INFO] Worker exiting (pid: 4982)
[2024-01-30 11:29:19 +0800] [4981] [ERROR] Worker (pid:4982) exited with code 3
[2024-01-30 11:29:19 +0800] [4981] [ERROR] Worker (pid:4983) was sent SIGTERM!
[2024-01-30 11:29:19 +0800] [4981] [ERROR] Shutting down: Master
[2024-01-30 11:29:19 +0800] [4981] [ERROR] Reason: Worker failed to boot.
[2024-01-30 11:31:08 +0800] [5025] [INFO] Starting gunicorn 21.2.0
[2024-01-30 11:31:08 +0800] [5025] [INFO] Listening at: http://0.0.0.0:9394 (5025)
[2024-01-30 11:31:08 +0800] [5025] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 11:31:08 +0800] [5026] [INFO] Booting worker with pid: 5026
[2024-01-30 11:31:08 +0800] [5027] [INFO] Booting worker with pid: 5027
[2024-01-30 11:31:08 +0800] [5026] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 5, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from scan_server.client import get_gvm_conn
ModuleNotFoundError: No module named 'scan_server'
[2024-01-30 11:31:08 +0800] [5026] [INFO] Worker exiting (pid: 5026)
[2024-01-30 11:31:08 +0800] [5025] [ERROR] Worker (pid:5026) exited with code 3
[2024-01-30 11:31:08 +0800] [5025] [ERROR] Worker (pid:5027) was sent SIGTERM!
[2024-01-30 11:31:08 +0800] [5025] [ERROR] Shutting down: Master
[2024-01-30 11:31:08 +0800] [5025] [ERROR] Reason: Worker failed to boot.
[2024-01-30 11:34:42 +0800] [5059] [INFO] Starting gunicorn 21.2.0
[2024-01-30 11:34:42 +0800] [5059] [INFO] Listening at: http://0.0.0.0:9394 (5059)
[2024-01-30 11:34:42 +0800] [5059] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 11:34:42 +0800] [5060] [INFO] Booting worker with pid: 5060
[2024-01-30 11:34:42 +0800] [5061] [INFO] Booting worker with pid: 5061
[2024-01-30 11:34:43 +0800] [5060] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 5, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from scan_server.client import get_gvm_conn
ModuleNotFoundError: No module named 'scan_server'
[2024-01-30 11:34:43 +0800] [5060] [INFO] Worker exiting (pid: 5060)
[2024-01-30 11:34:43 +0800] [5059] [ERROR] Worker (pid:5060) exited with code 3
[2024-01-30 11:34:43 +0800] [5059] [ERROR] Worker (pid:5061) was sent SIGTERM!
[2024-01-30 11:34:43 +0800] [5059] [ERROR] Shutting down: Master
[2024-01-30 11:34:43 +0800] [5059] [ERROR] Reason: Worker failed to boot.
[2024-01-30 11:35:22 +0800] [5080] [INFO] Starting gunicorn 21.2.0
[2024-01-30 11:35:22 +0800] [5080] [INFO] Listening at: http://0.0.0.0:9394 (5080)
[2024-01-30 11:35:22 +0800] [5080] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 11:35:22 +0800] [5081] [INFO] Booting worker with pid: 5081
[2024-01-30 11:35:22 +0800] [5082] [INFO] Booting worker with pid: 5082
[2024-01-30 11:35:23 +0800] [5081] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 5, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from scan_server.client import get_gvm_conn
ModuleNotFoundError: No module named 'scan_server'
[2024-01-30 11:35:23 +0800] [5081] [INFO] Worker exiting (pid: 5081)
[2024-01-30 11:35:23 +0800] [5082] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 5, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from scan_server.client import get_gvm_conn
ModuleNotFoundError: No module named 'scan_server'
[2024-01-30 11:35:23 +0800] [5082] [INFO] Worker exiting (pid: 5082)
[2024-01-30 11:35:23 +0800] [5080] [ERROR] Worker (pid:5081) exited with code 3
[2024-01-30 11:35:23 +0800] [5080] [ERROR] Worker (pid:5082) was sent SIGTERM!
[2024-01-30 11:35:23 +0800] [5080] [ERROR] Shutting down: Master
[2024-01-30 11:35:23 +0800] [5080] [ERROR] Reason: Worker failed to boot.
[2024-01-30 12:09:48 +0800] [5203] [INFO] Starting gunicorn 21.2.0
[2024-01-30 12:09:48 +0800] [5203] [INFO] Listening at: http://0.0.0.0:9394 (5203)
[2024-01-30 12:09:48 +0800] [5203] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 12:09:48 +0800] [5204] [INFO] Booting worker with pid: 5204
[2024-01-30 12:09:49 +0800] [5205] [INFO] Booting worker with pid: 5205
[2024-01-30 12:09:49 +0800] [5204] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 5, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from client import get_gvm_conn
  File "/home/gunicorn/scan_server/client.py", line 2, in <module>
    from pygvm.pygvm import Pygvm
  File "/home/gunicorn/scan_server/pygvm/pygvm.py", line 1, in <module>
    from utils import lxml_to_dict
ModuleNotFoundError: No module named 'utils'
[2024-01-30 12:09:49 +0800] [5204] [INFO] Worker exiting (pid: 5204)
[2024-01-30 12:09:49 +0800] [5203] [ERROR] Worker (pid:5204) exited with code 3
[2024-01-30 12:09:49 +0800] [5203] [ERROR] Worker (pid:5205) was sent SIGTERM!
[2024-01-30 12:09:49 +0800] [5203] [ERROR] Shutting down: Master
[2024-01-30 12:09:49 +0800] [5203] [ERROR] Reason: Worker failed to boot.
[2024-01-30 12:11:30 +0800] [5315] [INFO] Starting gunicorn 21.2.0
[2024-01-30 12:11:30 +0800] [5315] [INFO] Listening at: http://0.0.0.0:9394 (5315)
[2024-01-30 12:11:30 +0800] [5315] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 12:11:30 +0800] [5316] [INFO] Booting worker with pid: 5316
[2024-01-30 12:11:30 +0800] [5317] [INFO] Booting worker with pid: 5317
[2024-01-30 12:11:31 +0800] [5316] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 5, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from client import get_gvm_conn
  File "/home/gunicorn/scan_server/client.py", line 2, in <module>
    from pygvm.pygvm import Pygvm
  File "/home/gunicorn/scan_server/pygvm/pygvm.py", line 2, in <module>
    from .response import Response
  File "/home/gunicorn/scan_server/pygvm/response.py", line 9, in <module>
    from utils import lxml_to_dict
ModuleNotFoundError: No module named 'utils'
[2024-01-30 12:11:31 +0800] [5316] [INFO] Worker exiting (pid: 5316)
[2024-01-30 12:11:31 +0800] [5315] [ERROR] Worker (pid:5316) exited with code 3
[2024-01-30 12:11:31 +0800] [5315] [ERROR] Worker (pid:5317) was sent SIGTERM!
[2024-01-30 12:11:31 +0800] [5315] [ERROR] Shutting down: Master
[2024-01-30 12:11:31 +0800] [5315] [ERROR] Reason: Worker failed to boot.
[2024-01-30 12:12:47 +0800] [5405] [INFO] Starting gunicorn 21.2.0
[2024-01-30 12:12:47 +0800] [5405] [INFO] Listening at: http://0.0.0.0:9394 (5405)
[2024-01-30 12:12:47 +0800] [5405] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 12:12:47 +0800] [5406] [INFO] Booting worker with pid: 5406
[2024-01-30 12:12:47 +0800] [5407] [INFO] Booting worker with pid: 5407
[2024-01-30 12:12:49 +0800] [5406] [INFO] Started server process [5406]
[2024-01-30 12:12:49 +0800] [5406] [INFO] Waiting for application startup.
[2024-01-30 12:12:49 +0800] [5406] [INFO] Application startup complete.
[2024-01-30 12:12:49 +0800] [5407] [INFO] Started server process [5407]
[2024-01-30 12:12:49 +0800] [5407] [INFO] Waiting for application startup.
[2024-01-30 12:12:49 +0800] [5407] [INFO] Application startup complete.
[2024-01-30 12:13:18 +0800] [5405] [INFO] Handling signal: int
[2024-01-30 12:13:18 +0800] [5406] [INFO] Shutting down
[2024-01-30 12:13:18 +0800] [5407] [INFO] Shutting down
[2024-01-30 12:13:18 +0800] [5406] [INFO] Waiting for application shutdown.
[2024-01-30 12:13:18 +0800] [5406] [INFO] Application shutdown complete.
[2024-01-30 12:13:18 +0800] [5406] [INFO] Finished server process [5406]
[2024-01-30 12:13:18 +0800] [5406] [INFO] Worker exiting (pid: 5406)
[2024-01-30 12:13:18 +0800] [5407] [INFO] Waiting for application shutdown.
[2024-01-30 12:13:18 +0800] [5407] [INFO] Application shutdown complete.
[2024-01-30 12:13:18 +0800] [5407] [INFO] Finished server process [5407]
[2024-01-30 12:13:18 +0800] [5407] [INFO] Worker exiting (pid: 5407)
[2024-01-30 12:13:18 +0800] [5405] [INFO] Shutting down: Master
[2024-01-30 12:16:23 +0800] [5452] [INFO] Starting gunicorn 21.2.0
[2024-01-30 12:16:23 +0800] [5452] [INFO] Listening at: http://0.0.0.0:9394 (5452)
[2024-01-30 12:16:23 +0800] [5452] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 12:16:23 +0800] [5458] [INFO] Booting worker with pid: 5458
[2024-01-30 12:16:23 +0800] [5459] [INFO] Booting worker with pid: 5459
[2024-01-30 12:16:24 +0800] [5459] [INFO] Started server process [5459]
[2024-01-30 12:16:24 +0800] [5459] [INFO] Waiting for application startup.
[2024-01-30 12:16:24 +0800] [5459] [INFO] Application startup complete.
[2024-01-30 12:16:24 +0800] [5458] [INFO] Started server process [5458]
[2024-01-30 12:16:24 +0800] [5458] [INFO] Waiting for application startup.
[2024-01-30 12:16:24 +0800] [5458] [INFO] Application startup complete.
[2024-01-30 14:31:18 +0800] [5931] [INFO] Starting gunicorn 21.2.0
[2024-01-30 14:31:18 +0800] [5931] [INFO] Listening at: http://0.0.0.0:9394 (5931)
[2024-01-30 14:31:18 +0800] [5931] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-30 14:31:18 +0800] [5933] [INFO] Booting worker with pid: 5933
[2024-01-30 14:31:18 +0800] [5934] [INFO] Booting worker with pid: 5934
[2024-01-30 14:31:19 +0800] [5933] [INFO] Started server process [5933]
[2024-01-30 14:31:19 +0800] [5933] [INFO] Waiting for application startup.
[2024-01-30 14:31:19 +0800] [5933] [INFO] Application startup complete.
[2024-01-30 14:31:19 +0800] [5934] [INFO] Started server process [5934]
[2024-01-30 14:31:19 +0800] [5934] [INFO] Waiting for application startup.
[2024-01-30 14:31:19 +0800] [5934] [INFO] Application startup complete.
[2024-01-31 17:26:55 +0800] [234569] [INFO] Starting gunicorn 21.2.0
[2024-01-31 17:26:55 +0800] [234569] [INFO] Listening at: http://0.0.0.0:9394 (234569)
[2024-01-31 17:26:55 +0800] [234569] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-01-31 17:26:55 +0800] [234571] [INFO] Booting worker with pid: 234571
[2024-01-31 17:26:56 +0800] [234572] [INFO] Booting worker with pid: 234572
[2024-01-31 17:26:56 +0800] [234571] [INFO] Started server process [234571]
[2024-01-31 17:26:56 +0800] [234571] [INFO] Waiting for application startup.
[2024-01-31 17:26:56 +0800] [234571] [INFO] Application startup complete.
[2024-01-31 17:26:56 +0800] [234572] [INFO] Started server process [234572]
[2024-01-31 17:26:56 +0800] [234572] [INFO] Waiting for application startup.
[2024-01-31 17:26:56 +0800] [234572] [INFO] Application startup complete.
[2024-03-15 20:24:12 +0800] [416195] [INFO] Starting gunicorn 21.2.0
[2024-03-15 20:24:12 +0800] [416195] [INFO] Listening at: http://0.0.0.0:9394 (416195)
[2024-03-15 20:24:12 +0800] [416195] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-03-15 20:24:12 +0800] [416197] [INFO] Booting worker with pid: 416197
[2024-03-15 20:24:12 +0800] [416198] [INFO] Booting worker with pid: 416198
[2024-03-15 20:24:13 +0800] [416197] [INFO] Started server process [416197]
[2024-03-15 20:24:13 +0800] [416197] [INFO] Waiting for application startup.
[2024-03-15 20:24:13 +0800] [416197] [INFO] Application startup complete.
[2024-03-15 20:24:13 +0800] [416198] [INFO] Started server process [416198]
[2024-03-15 20:24:13 +0800] [416198] [INFO] Waiting for application startup.
[2024-03-15 20:24:13 +0800] [416198] [INFO] Application startup complete.
[2024-03-15 21:18:54 +0800] [417743] [INFO] Starting gunicorn 21.2.0
[2024-03-15 21:18:54 +0800] [417743] [INFO] Listening at: http://0.0.0.0:9394 (417743)
[2024-03-15 21:18:54 +0800] [417743] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-03-15 21:18:54 +0800] [417745] [INFO] Booting worker with pid: 417745
[2024-03-15 21:18:54 +0800] [417746] [INFO] Booting worker with pid: 417746
[2024-03-15 21:18:54 +0800] [417745] [INFO] Started server process [417745]
[2024-03-15 21:18:54 +0800] [417745] [INFO] Waiting for application startup.
[2024-03-15 21:18:54 +0800] [417745] [INFO] Application startup complete.
[2024-03-15 21:18:55 +0800] [417746] [INFO] Started server process [417746]
[2024-03-15 21:18:55 +0800] [417746] [INFO] Waiting for application startup.
[2024-03-15 21:18:55 +0800] [417746] [INFO] Application startup complete.
[2024-03-27 15:36:42 +0800] [557413] [INFO] Starting gunicorn 21.2.0
[2024-03-27 15:36:42 +0800] [557413] [INFO] Listening at: http://0.0.0.0:9394 (557413)
[2024-03-27 15:36:42 +0800] [557413] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-03-27 15:36:42 +0800] [557415] [INFO] Booting worker with pid: 557415
[2024-03-27 15:36:42 +0800] [557416] [INFO] Booting worker with pid: 557416
[2024-03-27 15:36:43 +0800] [557415] [INFO] Started server process [557415]
[2024-03-27 15:36:43 +0800] [557415] [INFO] Waiting for application startup.
[2024-03-27 15:36:43 +0800] [557415] [INFO] Application startup complete.
[2024-03-27 15:36:43 +0800] [557416] [INFO] Started server process [557416]
[2024-03-27 15:36:43 +0800] [557416] [INFO] Waiting for application startup.
[2024-03-27 15:36:43 +0800] [557416] [INFO] Application startup complete.
[2024-03-27 16:32:45 +0800] [608827] [INFO] Starting gunicorn 21.2.0
[2024-03-27 16:32:45 +0800] [608827] [INFO] Listening at: http://0.0.0.0:9394 (608827)
[2024-03-27 16:32:45 +0800] [608827] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-03-27 16:32:45 +0800] [608829] [INFO] Booting worker with pid: 608829
[2024-03-27 16:32:46 +0800] [608830] [INFO] Booting worker with pid: 608830
[2024-03-27 16:32:46 +0800] [608829] [INFO] Started server process [608829]
[2024-03-27 16:32:46 +0800] [608829] [INFO] Waiting for application startup.
[2024-03-27 16:32:46 +0800] [608829] [INFO] Application startup complete.
[2024-03-27 16:32:46 +0800] [608830] [INFO] Started server process [608830]
[2024-03-27 16:32:46 +0800] [608830] [INFO] Waiting for application startup.
[2024-03-27 16:32:46 +0800] [608830] [INFO] Application startup complete.
[2024-03-27 16:59:55 +0800] [646868] [INFO] Starting gunicorn 21.2.0
[2024-03-27 16:59:55 +0800] [646868] [INFO] Listening at: http://0.0.0.0:9394 (646868)
[2024-03-27 16:59:55 +0800] [646868] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-03-27 16:59:55 +0800] [646870] [INFO] Booting worker with pid: 646870
[2024-03-27 16:59:56 +0800] [646870] [INFO] Started server process [646870]
[2024-03-27 16:59:56 +0800] [646870] [INFO] Waiting for application startup.
[2024-03-27 16:59:56 +0800] [646870] [INFO] Application startup complete.
[2024-04-10 04:46:51 -0400] [11651] [INFO] Starting gunicorn 21.2.0
[2024-04-10 04:46:51 -0400] [11651] [INFO] Listening at: http://0.0.0.0:9394 (11651)
[2024-04-10 04:46:51 -0400] [11651] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-04-10 04:46:51 -0400] [11653] [INFO] Booting worker with pid: 11653
[2024-04-10 04:46:53 -0400] [11653] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/usr/lib/python3/dist-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
                ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
                    ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/gunicorn/scan_server/main.py", line 4, in <module>
    from routers import gvm_task, zap_task
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 2, in <module>
    from client import get_gvm_conn
  File "/home/gunicorn/scan_server/client.py", line 2, in <module>
    from pygvm.pygvm import Pygvm
  File "/home/gunicorn/scan_server/pygvm/pygvm.py", line 2, in <module>
    from .response import Response
  File "/home/gunicorn/scan_server/pygvm/response.py", line 16, in <module>
    from gvm.xml import pretty_print
  File "/usr/local/lib/python3.11/dist-packages/gvm/xml.py", line 23, in <module>
    import defusedxml.lxml as secET
ModuleNotFoundError: No module named 'defusedxml'
[2024-04-10 04:46:53 -0400] [11653] [INFO] Worker exiting (pid: 11653)
[2024-04-10 04:46:53 -0400] [11651] [ERROR] Worker (pid:11653) exited with code 3
[2024-04-10 04:46:53 -0400] [11651] [ERROR] Shutting down: Master
[2024-04-10 04:46:53 -0400] [11651] [ERROR] Reason: Worker failed to boot.
[2024-04-10 04:54:34 -0400] [16328] [INFO] Starting gunicorn 21.2.0
[2024-04-10 04:54:34 -0400] [16328] [INFO] Listening at: http://0.0.0.0:9394 (16328)
[2024-04-10 04:54:34 -0400] [16328] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-04-10 04:54:34 -0400] [16330] [INFO] Booting worker with pid: 16330
[2024-04-10 04:54:36 -0400] [16330] [INFO] Started server process [16330]
[2024-04-10 04:54:36 -0400] [16330] [INFO] Waiting for application startup.
[2024-04-10 04:54:36 -0400] [16330] [INFO] Application startup complete.
[2024-04-10 05:13:34 -0400] [29539] [INFO] Starting gunicorn 21.2.0
[2024-04-10 05:13:34 -0400] [29539] [INFO] Listening at: http://0.0.0.0:9394 (29539)
[2024-04-10 05:13:34 -0400] [29539] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-04-10 05:13:34 -0400] [29541] [INFO] Booting worker with pid: 29541
[2024-04-10 05:13:35 -0400] [29541] [INFO] Started server process [29541]
[2024-04-10 05:13:35 -0400] [29541] [INFO] Waiting for application startup.
[2024-04-10 05:13:35 -0400] [29541] [INFO] Application startup complete.
[2024-04-10 05:40:06 -0400] [42960] [INFO] Starting gunicorn 21.2.0
[2024-04-10 05:40:06 -0400] [42960] [INFO] Listening at: http://0.0.0.0:9394 (42960)
[2024-04-10 05:40:06 -0400] [42960] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-04-10 05:40:06 -0400] [42962] [INFO] Booting worker with pid: 42962
[2024-04-10 05:40:07 -0400] [42962] [INFO] Started server process [42962]
[2024-04-10 05:40:07 -0400] [42962] [INFO] Waiting for application startup.
[2024-04-10 05:40:07 -0400] [42962] [INFO] Application startup complete.
[2024-04-11 12:06:50 -0400] [938442] [INFO] Starting gunicorn 21.2.0
[2024-04-11 12:06:50 -0400] [938442] [INFO] Listening at: http://0.0.0.0:9394 (938442)
[2024-04-11 12:06:50 -0400] [938442] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-04-11 12:06:50 -0400] [938444] [INFO] Booting worker with pid: 938444
[2024-04-11 12:06:51 -0400] [938444] [INFO] Started server process [938444]
[2024-04-11 12:06:51 -0400] [938444] [INFO] Waiting for application startup.
[2024-04-11 12:06:51 -0400] [938444] [INFO] Application startup complete.
[2024-10-12 07:38:21 -0400] [860417] [INFO] Starting gunicorn 22.0.0
[2024-10-12 07:38:21 -0400] [860417] [INFO] Listening at: http://0.0.0.0:9394 (860417)
[2024-10-12 07:38:21 -0400] [860417] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-10-12 07:38:21 -0400] [860430] [INFO] Booting worker with pid: 860430
[2024-10-12 07:38:22 -0400] [860430] [INFO] Started server process [860430]
[2024-10-12 07:38:22 -0400] [860430] [INFO] Waiting for application startup.
[2024-10-12 07:38:22 -0400] [860430] [INFO] Application startup complete.
[2024-10-12 08:00:17 -0400] [860417] [CRITICAL] WORKER TIMEOUT (pid:860430)
[2024-10-12 08:00:17 -0400] [860417] [ERROR] Worker (pid:860430) was sent SIGABRT!
[2024-10-12 08:00:17 -0400] [871603] [INFO] Booting worker with pid: 871603
[2024-10-12 08:00:18 -0400] [871603] [INFO] Started server process [871603]
[2024-10-12 08:00:18 -0400] [871603] [INFO] Waiting for application startup.
[2024-10-12 08:00:18 -0400] [871603] [INFO] Application startup complete.
[2024-10-12 08:03:51 -0400] [860417] [CRITICAL] WORKER TIMEOUT (pid:871603)
[2024-10-12 08:03:51 -0400] [860417] [ERROR] Worker (pid:871603) was sent SIGABRT!
[2024-10-12 08:03:51 -0400] [873663] [INFO] Booting worker with pid: 873663
[2024-10-12 08:03:52 -0400] [873663] [INFO] Started server process [873663]
[2024-10-12 08:03:52 -0400] [873663] [INFO] Waiting for application startup.
[2024-10-12 08:03:52 -0400] [873663] [INFO] Application startup complete.
[2024-10-12 08:07:54 -0400] [860417] [CRITICAL] WORKER TIMEOUT (pid:873663)
[2024-10-12 08:07:54 -0400] [860417] [ERROR] Worker (pid:873663) was sent SIGABRT!
[2024-10-12 08:07:54 -0400] [876302] [INFO] Booting worker with pid: 876302
[2024-10-12 08:07:55 -0400] [876302] [INFO] Started server process [876302]
[2024-10-12 08:07:55 -0400] [876302] [INFO] Waiting for application startup.
[2024-10-12 08:07:55 -0400] [876302] [INFO] Application startup complete.
[2024-10-12 08:17:50 -0400] [882353] [INFO] Starting gunicorn 22.0.0
[2024-10-12 08:17:50 -0400] [882353] [INFO] Listening at: http://0.0.0.0:9394 (882353)
[2024-10-12 08:17:50 -0400] [882353] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-10-12 08:17:50 -0400] [882358] [INFO] Booting worker with pid: 882358
[2024-10-12 08:17:51 -0400] [882358] [INFO] Started server process [882358]
[2024-10-12 08:17:51 -0400] [882358] [INFO] Waiting for application startup.
[2024-10-12 08:17:51 -0400] [882358] [INFO] Application startup complete.
[2024-10-12 08:22:53 -0400] [882353] [CRITICAL] WORKER TIMEOUT (pid:882358)
[2024-10-12 08:22:53 -0400] [882353] [ERROR] Worker (pid:882358) was sent SIGABRT!
[2024-10-12 08:22:53 -0400] [885182] [INFO] Booting worker with pid: 885182
[2024-10-12 08:22:54 -0400] [885182] [INFO] Started server process [885182]
[2024-10-12 08:22:54 -0400] [885182] [INFO] Waiting for application startup.
[2024-10-12 08:22:54 -0400] [885182] [INFO] Application startup complete.
[2024-10-12 08:26:52 -0400] [887373] [INFO] Starting gunicorn 22.0.0
[2024-10-12 08:26:52 -0400] [887373] [INFO] Listening at: http://0.0.0.0:9394 (887373)
[2024-10-12 08:26:52 -0400] [887373] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-10-12 08:26:52 -0400] [887378] [INFO] Booting worker with pid: 887378
[2024-10-12 08:26:52 -0400] [887379] [INFO] Booting worker with pid: 887379
[2024-10-12 08:26:52 -0400] [887380] [INFO] Booting worker with pid: 887380
[2024-10-12 08:26:53 -0400] [887381] [INFO] Booting worker with pid: 887381
[2024-10-12 08:26:54 -0400] [887379] [INFO] Started server process [887379]
[2024-10-12 08:26:54 -0400] [887379] [INFO] Waiting for application startup.
[2024-10-12 08:26:54 -0400] [887379] [INFO] Application startup complete.
[2024-10-12 08:26:54 -0400] [887378] [INFO] Started server process [887378]
[2024-10-12 08:26:54 -0400] [887378] [INFO] Waiting for application startup.
[2024-10-12 08:26:54 -0400] [887378] [INFO] Application startup complete.
[2024-10-12 08:26:54 -0400] [887380] [INFO] Started server process [887380]
[2024-10-12 08:26:54 -0400] [887380] [INFO] Waiting for application startup.
[2024-10-12 08:26:54 -0400] [887380] [INFO] Application startup complete.
[2024-10-12 08:26:54 -0400] [887381] [INFO] Started server process [887381]
[2024-10-12 08:26:54 -0400] [887381] [INFO] Waiting for application startup.
[2024-10-12 08:26:54 -0400] [887381] [INFO] Application startup complete.
[2024-10-12 08:33:13 -0400] [890762] [INFO] Starting gunicorn 22.0.0
[2024-10-12 08:33:13 -0400] [890762] [INFO] Listening at: http://0.0.0.0:9394 (890762)
[2024-10-12 08:33:13 -0400] [890762] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-10-12 08:33:13 -0400] [890764] [INFO] Booting worker with pid: 890764
[2024-10-12 08:33:13 -0400] [890765] [INFO] Booting worker with pid: 890765
[2024-10-12 08:33:13 -0400] [890766] [INFO] Booting worker with pid: 890766
[2024-10-12 08:33:13 -0400] [890767] [INFO] Booting worker with pid: 890767
[2024-10-12 08:33:14 -0400] [890764] [INFO] Started server process [890764]
[2024-10-12 08:33:14 -0400] [890764] [INFO] Waiting for application startup.
[2024-10-12 08:33:14 -0400] [890764] [INFO] Application startup complete.
[2024-10-12 08:33:14 -0400] [890767] [INFO] Started server process [890767]
[2024-10-12 08:33:14 -0400] [890767] [INFO] Waiting for application startup.
[2024-10-12 08:33:14 -0400] [890767] [INFO] Application startup complete.
[2024-10-12 08:33:14 -0400] [890765] [INFO] Started server process [890765]
[2024-10-12 08:33:14 -0400] [890765] [INFO] Waiting for application startup.
[2024-10-12 08:33:14 -0400] [890765] [INFO] Application startup complete.
[2024-10-12 08:33:14 -0400] [890766] [INFO] Started server process [890766]
[2024-10-12 08:33:14 -0400] [890766] [INFO] Waiting for application startup.
[2024-10-12 08:33:14 -0400] [890766] [INFO] Application startup complete.
[2024-10-12 08:33:29 -0400] [890766] [ERROR] Exception in ASGI application
Traceback (most recent call last):
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 139, in create_task_with_config
    config_id = config_dict[config_name]
                ~~~~~~~~~~~^^^^^^^^^^^^^
TypeError: tuple indices must be integers or slices, not str

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/uvicorn/protocols/http/httptools_impl.py", line 399, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/uvicorn/middleware/proxy_headers.py", line 70, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/applications.py", line 123, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/middleware/errors.py", line 186, in __call__
    raise exc
  File "/usr/local/lib/python3.11/dist-packages/starlette/middleware/errors.py", line 164, in __call__
    await self.app(scope, receive, _send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/middleware/exceptions.py", line 65, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "/usr/local/lib/python3.11/dist-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "/usr/local/lib/python3.11/dist-packages/starlette/routing.py", line 756, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/routing.py", line 776, in app
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/routing.py", line 297, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/routing.py", line 77, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/usr/local/lib/python3.11/dist-packages/starlette/_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "/usr/local/lib/python3.11/dist-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "/usr/local/lib/python3.11/dist-packages/starlette/routing.py", line 72, in app
    response = await func(request)
               ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/fastapi/routing.py", line 278, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/fastapi/routing.py", line 191, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gunicorn/scan_server/routers/gvm_task.py", line 150, in create_task_with_config
    logger.error('Faild to create gvm task: ' + e.with_traceback)
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
TypeError: can only concatenate str (not "builtin_function_or_method") to str
[2024-10-12 08:42:50 -0400] [895894] [INFO] Starting gunicorn 22.0.0
[2024-10-12 08:42:50 -0400] [895894] [INFO] Listening at: http://0.0.0.0:9394 (895894)
[2024-10-12 08:42:50 -0400] [895894] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-10-12 08:42:50 -0400] [895896] [INFO] Booting worker with pid: 895896
[2024-10-12 08:42:50 -0400] [895902] [INFO] Booting worker with pid: 895902
[2024-10-12 08:42:50 -0400] [895906] [INFO] Booting worker with pid: 895906
[2024-10-12 08:42:50 -0400] [895907] [INFO] Booting worker with pid: 895907
[2024-10-12 08:42:51 -0400] [895906] [INFO] Started server process [895906]
[2024-10-12 08:42:51 -0400] [895906] [INFO] Waiting for application startup.
[2024-10-12 08:42:51 -0400] [895906] [INFO] Application startup complete.
[2024-10-12 08:42:51 -0400] [895902] [INFO] Started server process [895902]
[2024-10-12 08:42:51 -0400] [895902] [INFO] Waiting for application startup.
[2024-10-12 08:42:51 -0400] [895902] [INFO] Application startup complete.
[2024-10-12 08:42:51 -0400] [895907] [INFO] Started server process [895907]
[2024-10-12 08:42:51 -0400] [895907] [INFO] Waiting for application startup.
[2024-10-12 08:42:51 -0400] [895907] [INFO] Application startup complete.
[2024-10-12 08:42:51 -0400] [895896] [INFO] Started server process [895896]
[2024-10-12 08:42:51 -0400] [895896] [INFO] Waiting for application startup.
[2024-10-12 08:42:51 -0400] [895896] [INFO] Application startup complete.
[2024-10-12 09:09:26 -0400] [917818] [INFO] Starting gunicorn 22.0.0
[2024-10-12 09:09:26 -0400] [917818] [INFO] Listening at: http://0.0.0.0:9394 (917818)
[2024-10-12 09:09:26 -0400] [917818] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2024-10-12 09:09:26 -0400] [917820] [INFO] Booting worker with pid: 917820
[2024-10-12 09:09:26 -0400] [917821] [INFO] Booting worker with pid: 917821
[2024-10-12 09:09:26 -0400] [917822] [INFO] Booting worker with pid: 917822
[2024-10-12 09:09:26 -0400] [917823] [INFO] Booting worker with pid: 917823
[2024-10-12 09:09:29 -0400] [917823] [INFO] Started server process [917823]
[2024-10-12 09:09:29 -0400] [917823] [INFO] Waiting for application startup.
[2024-10-12 09:09:29 -0400] [917823] [INFO] Application startup complete.
[2024-10-12 09:09:29 -0400] [917822] [INFO] Started server process [917822]
[2024-10-12 09:09:29 -0400] [917822] [INFO] Waiting for application startup.
[2024-10-12 09:09:29 -0400] [917822] [INFO] Application startup complete.
[2024-10-12 09:09:29 -0400] [917821] [INFO] Started server process [917821]
[2024-10-12 09:09:29 -0400] [917821] [INFO] Waiting for application startup.
[2024-10-12 09:09:29 -0400] [917821] [INFO] Application startup complete.
[2024-10-12 09:09:29 -0400] [917820] [INFO] Started server process [917820]
[2024-10-12 09:09:29 -0400] [917820] [INFO] Waiting for application startup.
[2024-10-12 09:09:29 -0400] [917820] [INFO] Application startup complete.
